cfme/tests/test_replication.py

```diff
@@ -29,6 +31,24 @@ def create_vm(provider, vm_name):
     return vm
 
 
+def is_dicts_same(dict1, dict2):
```

Comment:
As per observation values are list. is it not possible to reduce code with `set` operation?